triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Server.scala

```diff
@@ -437,8 +437,18 @@ class Server(
         }
       }
     },
+    path("readyz") {
+      extractActorSystem { implicit sys =>
+        implicit val timeout: Timeout = Timeout(5 seconds)
+        implicit val scheduler: Scheduler = schedulerFromActorSystem(sys.toTyped)
+        onSuccess(ctx.self.ask(ref => Server.GetServerState(ref))) {
+          case Server.Running => complete(StatusCodes.OK)
+          case Server.Starting => complete(StatusCodes.NotFound)
+        }
+      }
+    },
     path("livez") {
-      complete((StatusCodes.OK, JsObject(("status", "pass".toJson))))
+      complete(StatusCodes.OK)
```

Comment:
Note that this is also a breaking change, if by any chance any user was inspecting the returned JSON. I would recommend rolling back this change unless it's necessary or add a deprecation. In any case, please report breaking changes on the change log or they will not be visible on the release notes.

Reply:
I realize I explicitly asked for the `/livez` endpoint to match the Kubernetes expected semantics. Without context around what Kubernetes does, I imagine this change was triggered by that request. If keeping the returned JSON doesn't break Kubernetes, I think it's find to leave it there and add a deprecation so that we can remove it in the future.

Reply:
I realize now that I got the change log entry the wrong way around. The `/livez` change is documented. What is lacking though, is the addition of the `/readyz` endpoint, which should be there as well.

Reply:
I reported the breaking change in the change log. The JSON body is simply not necessary.

Reply:
I'll also add the readyz endpoint to the change log

Reply:
@stefanobaghino-da the daml assistent integration tests failed because of the `/livez` change, so it's probably best to be more conservative as you suggested, I've left the `/livez` endpoint unchanged.